### ISSUE TITLE
Latest HA requires ac-infinity-ble 0.4.2 but fails to install it

### ISSUE THREAD

**Issue by douginoz** [hunterjm/ac-infinity-ble#7]
The problem is already well documented in https://github.com/hunterjm/ac-infinity-hacs so I'm just adding it here as a pointer.

The problem is that this version is missing and can't be installed manually if running HAoS (no access to the underlying os). There's no way to run pip or docker on HAOS (supervised).  SSH into it only grants access to a separate docker shell and not the underlying os. Then when pip is run, it spears to succeed. However, after rebooting HAOS the system log file shows that it isn't actually installed:

``2023-09-10 22:06:09.397 ERROR (MainThread) [homeassistant.setup] Setup failed for custom integration ac_infinity: Requirements for ac_infinity not found: ['ac-infinity-ble==0.4.2'].
homeassistant.requirements.RequirementsNotFound: Requirements for ac_infinity not found: ['ac-infinity-ble==0.4.2'].

**Comment by Kathou99** [comment]
I have the same issue

**Comment by AustinBrunkhorst** [comment]
[Here's a workaround](https://github.com/hunterjm/ac-infinity-hacs/issues/6#issuecomment-1721528670) until @hunterjm is available to create a new release.

> The problem is that this version is missing and can't be installed manually

FWIW, Home Assistant provides a misleading error message in this scenario. It's not that `ac-infinity-ble` isn't found, it's that it couldn't be installed due to dependency requirement conflicts. To uncover the useful error message, you have to run pip install with the `package_constraints.txt` from Home Assistant.

```
ERROR: Cannot install ac-infinity-ble==0.4.2 because these package versions have conflicting dependencies.

The conflict is caused by:
    ac-infinity-ble 0.4.2 depends on bleak<0.21.0 and >=0.20.2
    The user requested (constraint) bleak==0.21.1
```

**Comment by douginoz** [comment]
Resolved in ac-infinity-hacs 0.4.3

**Comment by douginoz** [comment]
Resolved in [ac-infinity-hacs 0.4.3](https://github.com/hunterjm/ac-infinity-hacs/releases/tag/v1.0.3)
